cmd/injector/main.go

```diff
@@ -90,29 +95,28 @@ func init() {
 	flag.StringVar(&credentials.RootCertFilename, "issuer-ca-secret-key", credentials.RootCertFilename, "Certificate Authority certificate secret key")
 	flag.StringVar(&credentials.IssuerCertFilename, "issuer-certificate-secret-key", credentials.IssuerCertFilename, "Issuer certificate secret key")
 	flag.StringVar(&credentials.IssuerKeyFilename, "issuer-key-secret-key", credentials.IssuerKeyFilename, "Issuer private key secret key")
+	flag.BoolVar(&metricsEnabled, "enable-metrics", false, "Metric enabled, default false")
```

Comment:
`"Metric enabled, default false"` -> `"Enable metrics"` (no need to say "default false" because it's implied with boolean flags)